.github/workflows/winget.yml

```diff
@@ -6,10 +6,9 @@ on:
 
 jobs:
   publish:
-    # Action can only be run on windows
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
     steps:
-      - uses: vedantmgoyal2009/winget-releaser@v2
+      - uses: vedantmgoyal9/winget-releaser@main
```

Comment:
Accout names are similar, did you change your GitHub account name recently?

Reply:
I'd changed my username.

Reply:
Thank you for confirmation.
> https://github.com/sqlitebrowser/sqlitebrowser/pull/3690#pullrequestreview-2195350872

I actually had one more question, but I think you may have missed it, so I'd appreciate it if you could check.